apps/parsley/src/components/DetailsMenu/DetailsMenuCard/Toggles/JumpToFailingLineToggle/index.tsx

```diff
@@ -16,19 +19,37 @@ const JumpToFailingLineToggle: React.FC<JumpToFailingLineToggleProps> = ({
   const { sendEvent } = usePreferencesAnalytics();
   const { logMetadata } = useLogContext();
   const isTaskLog = logMetadata?.logType === LogTypes.EVERGREEN_TASK_LOGS;
-
+  const { closeSecondGuideCue, isOpenSecondGuideCue, setIsOpenSecondGuideCue } =
+    useSectionsFeatureDiscoveryContext();
+  const triggerRef = useRef<HTMLDivElement>(null);
   return (
-    <BaseToggle
-      data-cy="jump-to-failing-line-toggle"
-      disabled={!isTaskLog}
-      label="Jump to Failing Line"
-      onChange={(value) => {
-        updateSettings({ jumpToFailingLineEnabled: value });
-        sendEvent({ name: "Toggled jump to failing line", on: value });
-      }}
-      tooltip="Automatically scroll to the failing log line on page load. Only available for task logs."
-      value={checked}
-    />
+    <>
+      <GuideCue
+        beaconAlign="right"
+        data-cy="sections-cue-2"
+        numberOfSteps={1}
```

Comment:
The multi-step guide cue had issues with bouncing around when the the details menu loaded. Also, the blue beacon dot did not look great.